Dockerfiles/Dockerfile.init

```diff
@@ -3,15 +3,17 @@
 # some arguments that must be supplied
 ARG BASEIMAGE
 
+FROM amaas-eos-mw1.cec.lab.emc.com:5106/dcm/go-builder-image:latest as builder
+WORKDIR /upgrade
+COPY scripts/upgrade ./
+RUN wget https://dl.k8s.io/release/v1.26.4/bin/linux/amd64/kubectl -O kubectl
+
+
 FROM $BASEIMAGE AS init
 WORKDIR /upgrade
 COPY scripts/upgrade ./
-RUN microdnf update -y && \
-    microdnf install -y shadow-utils wget && \
-    wget -v https://dl.k8s.io/release/v1.25.7/bin/linux/amd64/kubectl && \
-    microdnf -y remove shadow-utils wget && \
-    microdnf clean all && \
+RUN mkdir -p ./kubectl /usr/local/bin && \
```

Comment:
This line means to create two directories. Don't understand why you are creating a directory ./kubectl.